packages/ra-ui-materialui/package.json

```diff
@@ -42,7 +42,7 @@
         "react": "^16.9.0",
         "react-dom": "^16.9.0",
         "react-final-form": "^6.5.0",
-        "react-final-form-arrays": "^3.1.1",
+        "react-final-form-arrays": "^3.1.2",
```

Comment:
You should run `make install` (which runs `yarn install`) in order to upgrade the `yarn.lock` file. It's the only way to really upgrade a dependency.

See [Yarn documentation](https://yarnpkg.com)